campaignion_layout/src/Lookup.php

```diff
@@ -71,4 +71,19 @@ public function getLayout() {
     return $this->themes->getTheme()->getLayoutFromItems($this->iterateItems());
   }
 
+  /**
+   * Get the order for the form for the currently active layout.
+   *
+   * @return string|null
+   *   The machine name of the ordering.
+   */
+  public function getOrder() {
```

Comment:
I don’t think this makes any sense wrt multiple items. This might choose one item for the layout and then another one for the order. Maybe we should switch to returning the item instead and then reading all its options?

Reply:
Different items how? It's checking for the layout name and if the layout enables the ordering...

Reply:
> Different items how?

Can you guarantee that there is only one field item with the said layout name?

Anyway getting the item only once is prettier I think.

Reply:
Sorry. I do not get the items stuff here. Why is it different to `getTheme()` or `getLayoutFromItems()` which also return the first matching item. I thought this is how the layout stuff is allowed to work... :thinking:

Reply:
`getTheme()` and `getLayoutFromItems()` run at different times. `getLayoutFromItems()` and `getOrder()` would both run during page preprocessing.

Reply:
Implemented my own suggestion.